### ISSUE TITLE
Install error with V0.4.0 : Cannot find module '../utils'

### ISSUE THREAD

**Issue by philohelp** [mantrajs/mantra-cli#94]
```
mantra g component core:myEditor -c
module.js:327
    throw err;
    ^

Error: Cannot find module '../utils'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/mantra-cli/dist/cli.js:13:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
```

I had to downgrade to 0.4.0-rc.4 then everything works again.

**Comment by terry-fei** [comment]
me too

**Comment by darkship** [comment]
In cli.js
<code>import {readConfig} from '../utils';</code> should be <code>import {readConfig} from './utils';</code>

**Comment by sungwoncho** [comment]
@darkship could you do a PR?

**Comment by ghost** [comment]
Hi @sungwoncho , I created a PR for this one. Thanks.

**Comment by sungwoncho** [comment]
Fixed with #95. Released in `0.4.1`.
